src/main/java/au/com/dmg/fusion/request/paymentrequest/extenstiondata/ExtensionData.java

```diff
@@ -32,38 +32,46 @@ public class ExtensionData {
     @Json(name = "TransitData")
     private final TransitData transitData;
 
-    @NotNull
+    @Json(name = "POIInformation")
```

Comment:
POIInformation should not be inside the existing class ExtensionData (which contains TransitData).  It should be inside another _ExtensionData_ class named "**ResponseExtensionData**".